NguyenVanViet2004/FE-GlowUp
# Implement booking detail UI

## Related issues
Resolved #26 

## Summary
- Implement Booking screen includes: 
   + Dark mode, light mode
- Update translation key 

## How to test
Step1: Change BASE_URL in HttpClient.ts file with your ipv4
Step2:
- Run the app and log in to your account (make sure your account has booking data or if your account does not have it, please log in with the following account {account: 0327367649, password: 123456})
- Click bottomTab Booking

## Concern points

## Screenshots
| Light mode | Dark mode |
| --- | --- |
|![image](https://github.com/user-attachments/assets/7bdbe69d-caba-4839-bfa0-a157e5046064)|![image](https://github.com/user-attachments/assets/06b8f49c-0bd5-4476-98a1-ef8ed5d5aac0)|![image](https://github.com/user-attachments/assets/837d664b-5b29-4aa9-b0ec-43f85d14aca8)|
![image](https://github.com/user-attachments/assets/2e8149d6-2d62-414d-84e5-8942658d903a)|![image](https://github.com/user-attachments/assets/7a0f5071-8222-4c71-b52c-ffcfa0c2bd45)|![image](https://github.com/user-attachments/assets/bcd0e2b3-355b-4979-88cf-5550d35c53c5)|
